plugin-server/src/utils/db/hub.ts

```diff
@@ -207,8 +207,9 @@ export async function createHub(
     status.info('👍', `Redis ready`)
 
     status.info('🤔', `Connecting to object storage...`)
+    let objectStorage = noOpStorage
     try {
-        connectObjectStorage(serverConfig)
+        objectStorage = connectObjectStorage(serverConfig)
```

Comment:
This try-catch is dead code - let's streamline all of this and remove the status.infos/try-catches etc.